### ISSUE TITLE
ArrayOutOfBoundsException using BruteForcePacker

### ISSUE THREAD

**Issue by othp** [skjolber/3d-bin-container-packing#11]
The following test will fail with an ArrayOutOfBoundsException:
``` 
	@Test
	public void testBlowsUpWithArrayOutOfBounds() {
		List<Dimension> containers = Arrays.asList(
			new Dimension("2", 330, 222, 121),
			new Dimension("4", 330, 235, 225)
		);

		List<BoxItem> items = Arrays.asList(
			new BoxItem(new Box(105, 105, 293), 1),
			new BoxItem(new Box(92, 94, 255), 1),
			new BoxItem(new Box(105, 70, 60), 2)
		);

		BruteForcePackager packer = new BruteForcePackager(containers);
		packer.pack(items);
	}
```

It looks like in the `PermutationRotationIterator` class, `nextRotation` is setting the rotation array differently from the way that its used in the `get` and `isWithinHeight` methods.

**Comment by skjolber** [comment]
See https://github.com/skjolber/3d-bin-container-packing/pull/12

**Comment by skjolber** [comment]
Released as version 1.0.7
